internal/datanode/channel_meta.go

```diff
@@ -294,13 +311,39 @@ func (c *ChannelMeta) filterSegments(partitionID UniqueID) []*Segment {
 }
 
 func (c *ChannelMeta) InitPKstats(ctx context.Context, s *Segment, statsBinlogs []*datapb.FieldBinlog, ts Timestamp) error {
+	if Params.DataNodeCfg.SkipBFStatsLoad {
+		// mark segment lazy loading
+		s.setLoadingLazy(true)
+		go func() {
```

Comment:
didn't really make any sense to me
Why we go the function out but mean time we use a pool?

Reply:
removed

Reply:
Recalled why add a goroutine to submit task into the work pool: if the worker pool is full, the Submit call will block here until there are some available worker.